trunk/NDHMS/Routing/module_HYDRO_io.F

```diff
@@ -8885,6 +8908,8 @@ subroutine read_route_link_netcdf( route_link_file,                         &
 where(So .lt. 0.0001) So=0.00001
 call get_1d_netcdf_real(ncid, 'ChSlp',    ChSSlp,    'read_route_link_netcdf', .TRUE.)
 call get_1d_netcdf_real(ncid, 'BtmWdth',  Bw,        'read_route_link_netcdf', .TRUE.)
+!! Loads channel infiltration, by default is zero, my need to add namelist option in future
+call get_1d_netcdf_real(ncid, 'Kchan',  ChannK,     'read_route_link_netcdf', .TRUE.)
```

Comment:
No issues with this line now. In the future, we could put it in an if statement in the future, if channel loss is a namelist option. Otherwise, we could set ChannK to zero.